src/game.cpp

```diff
@@ -7392,6 +7395,14 @@ look_around_result game::look_around( catacurses::window w_info, tripoint &cente
             }
         } else if( action == "TIMEOUT" ) {
             blink = !blink;
+        } else if( action == "fire" ) {
+            u.last_target_pos = g->m.getabs( lp );
+            plfire( u.weapon );
```

Comment:
Hadoken! [shoots a fireball from one's fists].

Reply:
I wish! `plfire()` is smart enough to tell you that "The none [hah!] can't be fired in its current state" so I guess you need to acquire a martial art with Hadoken first. :)

On a more serious note, does this need revision?

Reply:
I'll probably make `plfire()` say "You are not wielding any weapon" in case your hands are free but let me know if there's anything else.

Reply:
Obsolete due to commit acc9f5b3c1a11f4fc812c9b2058e422c5c4b4290.

Reply:
I'd leave these particular changes for another PR: they're unrelated to scrolling, whilst being pretty extensive, and require thorough testing.

Reply:
Ack, will do that once it's clear how I should proceed with the rest.